libraries/AP_Airspeed/AP_Airspeed_Health.cpp

```diff
@@ -37,7 +37,9 @@ void AP_Airspeed::check_sensor_ahrs_wind_max_failures(uint8_t i)
     }
 
     const AP_GPS &gps = AP::gps();
-    if (gps.status() < AP_GPS::GPS_Status::GPS_OK_FIX_3D) {
+    const uint8_t gps_instance = gps.primary_sensor();
+    const auto gps_status = gps.status(gps_instance);
+    if (gps_status < AP_GPS::GPS_Status::GPS_OK_FIX_3D) {
```

Comment:
This is the same as before. I guess its slightly clearer that were using primary, not sure its worth it tho.

Reply:
The point is, in the very near future, I am going to propose using fused GPS speed.

Reply:
IE: We do explicit checks of both airspeed sensors, why do we only compare to the output of one GPS?

Reply:
This can go in that PR then ;)

My comment about that is that if there is a more optimal fused speed available we should use that all over and not just in airspeed. If you do have blended GPS setup that becomes primary.

Reply:
Got it. I will take these changes out too.

Reply:
Done